app/forms/state_file/az_state_credits_form.rb

```diff
@@ -10,6 +10,14 @@ def save
       attributes_to_save[:tribal_wages] = nil if tribal_member == "no"
       attributes_to_save[:armed_forces_wages] = nil if armed_forces_member == "no"
       @intake.update(attributes_to_save)
+
+      # Create submission now so PDF link button can be shown on the next page
+      efile_submission = EfileSubmission.create!(
+        data_source: @intake,
+        )
+      if Rails.env.development? || Rails.env.test?
+        efile_submission.transition_to(:preparing)
+      end
```

Comment:
Didn't seem possible to construct a link to the PDF on the next screen unless I create the submission first. So I took out the "Submit" page and moved the creation code in here. Looks like it's ok to create it early and not advance it in the state machine flow, but i could be missing something.

Reply:
maybe it would be better to at least move this into AzReviewContoller::edit in case the page order changes

Reply:
I'm a little wary of creating it here or on the review page's `#edit`

In CTC-land, we created it on the "confirm legal" page when people did their final sign-off https://github.com/codeforamerica/vita-min/blob/main/app/forms/ctc/confirm_legal_form.rb#L31

From the figma, they haven't actually filled out their Direct Deposit details or signed that PDF yet so it's technically incomplete. I'm wary of including a link to download a PDF at this point in the flow, but if it is meant to be somewhat incomplete, maybe we could find a way to generate it that doesn't require a persisted EfileSubmission?

Might require a lot of architectural change, maybe worth putting off till later.

Reply:
I agree with Travis. I think we should remove the link for now, or just use a fake one, and perhaps have a follow up story for generating a PDF preview. We definitely don't want to send off the submission at this stage, which I assume `EfileSubmission.create!` would do.

Reply:
i can look into what it would take to show the pdf without creating a submission first.

pretty sure the point of adding the pdf here was to make this simpler for engineering :) the previous design was displaying a bunch of federal return data instead, which wouldn't actually be that hard, so i can talk to design about that option too.

Reply:
lauren agreed to move the pdf viewing later in the flow, and we just replaced that button with a short message telling the user they'll be able to review the completed pdf later. easy peasy.